pkgs/development/python-modules/mat2/default.nix

```diff
@@ -50,18 +50,6 @@ buildPythonPackage rec {
     ./executable-name.patch
     # hardcode path to mat2 executable
     ./tests.patch
-    # remove for next release
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
https://github.com/NixOS/nixpkgs/pull/138196